.github/workflows/node.js.yml

```diff
@@ -26,18 +26,19 @@ jobs:
       - run: yarn
       - run: yarn test
 
-  # windows:
-  #   runs-on: windows-latest
-
-  #   strategy:
-  #     matrix:
-  #       node-version: [14.x]
-
-  #   steps:
-  #     - uses: actions/checkout@v2
-  #     - name: Use Node.js ${{ matrix.node-version }} with windows
-  #       uses: actions/setup-node@v1
-  #       with:
-  #         node-version: ${{ matrix.node-version }}
-  #     - run: yarn
-  #     - run: yarn test
+  windows:
+    runs-on: windows-latest
+
+    strategy:
+      matrix:
+        node-version: [14.x]
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Use Node.js ${{ matrix.node-version }} with windows
+        uses: actions/setup-node@v1
```

Comment:
You might want to update to `v2`, https://github.com/actions/setup-node/releases/tag/v2.1.5 it brings some beneftis.